root/release/edit/tracklist.tt

```diff
@@ -338,6 +338,23 @@
        %]
     </table>
 
+    <!-- ko if: hasUselessMediumTitle -->
+      <div class="medium-title warning">
+        [%- warning_icon %]
+        <p>
+          <strong>[% l('Warning:') | html_entity %]</strong>
+          [% l('“CD X” / “Disc X” should only be used as a medium title if it’s the actual title for the medium (not just to indicate this is the disc number X in a set).') %]
+        </p>
+        <label>
+          <input id="confirm-medium-title" type="checkbox" data-bind="checked: confirmedMediumTitle" />
+          <span id="confirm-medium-title-message" data-bind="text: confirmMediumTitleMessage()" />
+        </label>
+        <p class="error field-error" style="padding-left: 1em" data-bind="showErrorWhenTabIsSwitched: hasUnconfirmedUselessMediumTitle">
+          [%~ l('If you’re sure the entered medium title is correct, confirm it above. Otherwise, please fix the data.') %]
```

Comment:
As an editor, from this warning I would not understand when to do it or not do it. A more detailed documentation with examples would be helpful. I asked for real examples in comment to the ticket.

Reply:
The place for the detailed documentation is [https://musicbrainz.org/doc/Style/Release#Medium_title]. Maybe link to it?

Also, if not entering “CD X”, editor may think that something is amiss. Maybe explain that numbering should just be set by moving mediums using up/down arrows?

Reply:
So, something like this instead of the first section?

> “CD X” / “Disc X” should only be used as a medium title if it’s the actual title for the medium (see {release_style|the guidelines}). To indicate ordering, simply reorder the medium using the up/down arrows.

Reply:
Trying something:
* Exposing the issue first, then possible courses of action;
* Displaying matched part of the title can be displayed instead of “CD X / Disc X”.

> “{matched_text}” seems to indicate medium ordering rather than medium title. If that is the case, please use up/down arrows on the right side to set medium position instead of title (see {release_style|the guidelines}). Otherwise please confirm that is not the case using the checkbox below.

Reply:
that sounds good to me but I'd tweak the grammar/wording slightly
> “{matched_text}” seems to indicate medium ordering rather than a medium title. If this is the case, please use the up/down arrows on the right side to set the medium position instead of adding a title (see {release_style|the guidelines}). Otherwise, please confirm that this is the actual title using the checkbox below.

Reply:
IIRC the reason I didn't do `{matched_text}` originally was to avoid having to create this in another page as with `confirmMediumTitleMessage()` (@mwiencek suggested it wasn't needed in both places). But I think with this wording it does make sense to have it, so I'll look into it.

Reply:
I think it works fine, so updating with it:

![Screenshot from 2021-11-17 20-51-05](https://user-images.githubusercontent.com/1069224/142263791-3456a1f3-2227-4ef0-b3ee-00ebe1dff30c.png)